Makefile

```diff
@@ -1856,18 +1863,18 @@ endif
 install-compiler-sources:
 ifeq "$(INSTALL_SOURCE_ARTIFACTS)" "true"
 	$(INSTALL_DATA) \
-	   utils/*.ml parsing/*.ml typing/*.ml bytecomp/*.ml driver/*.ml \
-           file_formats/*.ml \
-           lambda/*.ml \
-	   toplevel/*.ml toplevel/byte/*.ml \
-	   middle_end/*.ml middle_end/closure/*.ml \
-     middle_end/flambda/*.ml middle_end/flambda/base_types/*.ml \
-	   asmcomp/*.ml \
-	   asmcmp/debug/*.ml \
+	   $(SRCDIR)/utils/*.ml $(SRCDIR)/parsing/*.ml $(SRCDIR)/typing/*.ml $(SRCDIR)/bytecomp/*.ml $(SRCDIR)/driver/*.ml \
+           $(SRCDIR)/file_formats/*.ml \
+           $(SRCDIR)/lambda/*.ml \
+	   $(SRCDIR)/toplevel/*.ml $(SRCDIR)/toplevel/byte/*.ml \
+	   $(SRCDIR)/middle_end/*.ml $(SRCDIR)/middle_end/closure/*.ml \
+     $(SRCDIR)/middle_end/flambda/*.ml $(SRCDIR)/middle_end/flambda/base_types/*.ml \
+	   $(SRCDIR)/asmcomp/*.ml \
+	   $(SRCDIR)/asmcmp/debug/*.ml \
```

Comment:
I think you should break the list to have only one item per line for readability. What do we do for generated `.ml` files?